.travis.yml

```diff
@@ -103,9 +98,13 @@ script:
     - install_run $PLAT
 
 after_success:
-    # Upload wheels to Rackspace container
-    - pip install wheelhouse-uploader
-    - python -m wheelhouse_uploader upload --local-folder
-          ${TRAVIS_BUILD_DIR}/wheelhouse/
-          --no-update-index
-          wheels
+    # Upload the generated wheel package to anaconda.org
+    # PYWAVELETS_STAGING_UPLOAD_TOKEN is an encrypted variable
+    # used in Appveyor CI config, originally created at
+    # multibuild-wheels-staging site
+    - ANACONDA_ORG="multibuild-wheels-staging";
+      TOKEN=${PYWAVELETS_STAGING_UPLOAD_TOKEN};
```

Comment:
missing - ?
```suggestion
    - TOKEN=${PYWAVELETS_STAGING_UPLOAD_TOKEN};
```

Reply:
hmm, that seems to work as-is on astropy. I wonder why travis is not running?

Reply:
It seems to have just not liked that the root Python was set to 3.5.  Updating that to 3.6 (and updating multibuild itself) resolved the issue.